plugins/autolink/plugin.js

```diff
@@ -9,11 +9,14 @@
 	var doubleQuoteRegex = /"/g;
 
 	CKEDITOR.plugins.add( 'autolink', {
-		requires: 'clipboard,textmatch',
+		requires: 'clipboard,textmatch,link',
 
 		init: function( editor ) {
-			var urlTemplate = new CKEDITOR.template( '<a href="{link}">{text}</a>' ),
-				emailTemplate = new CKEDITOR.template( '<a href="mailto:{link}">{text}</a>' );
+
+			// (#2208)
+			if ( !this.isSupportedEnvironment() ) {
```

Comment:
With this early return added we should remove lines 39-42 as they duplicate the same logic.

Reply:
Duplicated logic removed: https://github.com/ckeditor/ckeditor4/pull/2518/commits/246502b71b98ea5d4fec093d0044256dcf611956